.storybook/theme.js

```diff
@@ -0,0 +1,36 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  colorPrimary: '#e95c30',
+  colorSecondary: '#e95c30',
+
+  // UI
+  appBg: '#555770',
+  appContentBg: 'white',
+  appBorderColor: 'white',
+  appBorderRadius: 4,
+
+  // Typography
+  fontBase: '"Open Sans", sans-serif',
```

Comment:
I spent a while trying to 'fix' this and bring in a better font.. but have given up.